src/test/java/org/broadinstitute/hellbender/engine/FeatureManagerUnitTest.java

```diff
@@ -188,6 +189,33 @@ public void testHandleRequestForValidFeatureInputs() {
     }
 
     @Test
+    public void testSequenceDictionary() {
+        final ValidFeatureArgumentSource toolInstance = new ValidFeatureArgumentSource();
+
+        // Initialize two of the FeatureInput fields as they would be initialized by the argument-parsing
```

Comment:
This comment is inaccurate -- you are only initializing one of the `FeatureInput` fields here, aren't you?